src/main/groovy/wooga/gradle/xcodebuild/XcodeBuildPlugin.groovy

```diff
@@ -44,20 +44,27 @@ class XcodeBuildPlugin implements Plugin<Project> {
     @Override
     void apply(Project project) {
         this.project = project
-        def extension = project.extensions.create(XcodeBuildPluginExtension, EXTENSION_NAME, DefaultXcodeBuildPluginExtension, project)
-        def tasks = project.tasks
 
+        def extension = project.extensions.create(XcodeBuildPluginExtension, EXTENSION_NAME, DefaultXcodeBuildPluginExtension, project)
         project.pluginManager.apply(BasePlugin.class)
 
+        configureExtension(extension, project)
+        def archives = project.configurations.maybeCreate('archives')
+        project.configurations['default'].extendsFrom(archives)
+        configureTasks(project, extension)
+    }
+
+    private static void configureExtension(XcodeBuildPluginExtension extension, Project project) {
```

Comment:
where is this coming from?

Reply:
Hmm? There were several invocations which I refactored onto the configureExtension method.